### ISSUE TITLE
Sensor init fails when using an I2C muxer

### ISSUE THREAD

**Issue by caternuson** [adafruit/Adafruit_CircuitPython_MLX90393#37]
Re this thread:
https://forums.adafruit.com/viewtopic.php?t=209506

```python
Adafruit CircuitPython 8.2.6 on 2023-09-12; Adafruit QT Py RP2040 with rp2040
>>> import board
>>> import adafruit_mlx90393
>>> import adafruit_tca9548a
>>> i2c = board.STEMMA_I2C()
>>> mux = adafruit_tca9548a.PCA9546A(i2c)
>>> SENSOR = adafruit_mlx90393.MLX90393(mux[2], address=0x0c)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "adafruit_mlx90393.py", line 226, in __init__
  File "adafruit_mlx90393.py", line 318, in resolution_x
  File "adafruit_mlx90393.py", line 351, in _set_resolution
  File "adafruit_mlx90393.py", line 417, in read_reg
  File "adafruit_tca9548a.py", line 74, in readfrom_into
OSError: [Errno 19] No such device
>>> 
```

**Comment by caternuson** [comment]
The read that is failing is getting NAK'd:
![image](https://github.com/adafruit/Adafruit_CircuitPython_MLX90393/assets/8755041/3cce9aba-aefb-4d4f-b959-fca2cfd59da3)


For comparison, here is the same thing **without** a muxer:
```python
Adafruit CircuitPython 8.2.6 on 2023-09-12; Adafruit QT Py RP2040 with rp2040
>>> import board
>>> import adafruit_mlx90393
>>> i2c = board.STEMMA_I2C()
>>> SENSOR = adafruit_mlx90393.MLX90393(i2c, address=0x0c)
>>> 
```
![image](https://github.com/adafruit/Adafruit_CircuitPython_MLX90393/assets/8755041/cc7e8fdf-0a24-4fc1-a29d-08566185fe75)

**Comment by caternuson** [comment]
Closing. Reported as having been fixed by #38.
https://forums.adafruit.com/viewtopic.php?p=1009613#p1009613
